lessons/de/chapter_5.yaml

```diff
@@ -0,0 +1,224 @@
+- title: Dereferenzierung
+  content_markdown: >
+    Mithilfe von '&mut'-Referenzen können Sie den Wert des Eigentümers mithilfe des '*'' festlegen
+    Operator.
+
+
+    Sie können auch eine Kopie eines eigenen Werts erhalten, indem Sie den Operator '*'' verwenden (wenn der Wert
+    können kopiert werden (wir werden kopierbare Typen in späteren Kapiteln besprechen).
+  code: >-
+    https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=fn%20main()%20%7B%0A%20%20%20%20let%20mut%20foo%20%3D%2042%3B%0A%20%20%20%20let%20f%20%3D%20%26mut%20foo%3B%0A%20%20%20%20let%20bar%20%3D%20*f%3B%20%2F%2F%20primim%20o%20copie%20a%20valorii%20proprietarului%0A%20%20%20%20*f%20%3D%2013%3B%20%20%20%20%20%20%2F%2F%20set%C4%83m%20valoarea%20proprietarului%20prin%20referin%C8%9B%C4%83%0A%20%20%20%20println!(%22%7B%7D%22%2C%20bar)%3B%0A%20%20%20%20println!(%22%7B%7D%22%2C%20foo)%3B%0A%7D%0A
+- title: Weitergabe geliehener Daten
+  content_markdown: >
+    Rusts Regeln für Referenzen lassen sich am besten wie folgt zusammenfassen:
+
+    * Rust erlaubt nur eine veränderbare Referenz **oder** mehrere
+    nicht veränderliche Referenzen **aber nicht beides**.
+
+    * Eine Referenz darf niemals **länger leben** als ihr Besitzer.Eine Referenz darf niemals **länger leben** als ihr Besitzer.
+
+
+    Dies stellt in der Regel kein Problem dar, wenn Verweise weitergegeben werden Funktionen.
+
+
+    Speicherdetails:
+
+    * Die erste Referenzregel verhindert Datenwettläufe. Was ist ein Datenwettlauf? A
+    Beim Datenwettlauf kann es beim Lesen von Daten zu Abweichungen kommen
+    aufgrund der gleichzeitigen Existenz eines Schreibers für die Daten. Das passiert
+    häufig in der Multithread-Programmierung.
+
+    * Die zweite Referenzregel verhindert den Missbrauch von Referenzen, die verweisen
+    auf nicht vorhandene Daten (in C baumelnde Zeiger genannt).
+  code: >-
+    https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=struct%20Foo%20%7B%0A%20%20%20%20x%3A%20i32%2C%0A%7D%0A%0Afn%20do_something(f%3A%20%26mut%20Foo)%20%7B%0A%20%20%20%20f.x%20%2B%3D%201%3B%0A%20%20%20%20%2F%2F%20se%20renun%C8%9B%C4%83%20la%20referin%C8%9Ba%20mutabil%C4%83%20f%20aici%0A%7D%0A%0Afn%20main()%20%7B%0A%20%20%20%20let%20mut%20foo%20%3D%20Foo%20%7B%20x%3A%2042%20%7D%3B%0A%20%20%20%20do_something(%26mut%20foo)%3B%0A%20%20%20%20%2F%2F%20pentru%20c%C4%83%20se%20renun%C8%9B%C4%83%20la%20toate%20referin%C8%9Bele%20mutabile%0A%20%20%20%20%2F%2F%20%C3%AEn%20cadrul%20func%C8%9Biei%20do_something%2C%20putem%20crea%20alta%0A%20%20%20%20do_something(%26mut%20foo)%3B%0A%20%20%20%20%2F%2F%20se%20renun%C8%9B%C4%83%20la%20foo%20aici%0A%7D%0A
```

Comment:
Please translate the code to German or use the English version instead of the Romanian version.